clip/requirements.txt

```diff
@@ -3,4 +3,5 @@ numpy
 transformers
 torch
 huggingface_hub
-Pillow
\ No newline at end of file
+Pillow
+mlx-lm
```

Comment:
I would prefer not to make `mlx-lm` a dep of this package. It might make things a little too brittle

Reply:
It's just for `save_weights`, so I would just copy a possibly simplified version of that here.

Reply:
yeah, make sense